test/Microsoft.Azure.ServiceBus.UnitTests/ExpectedMessagingExceptionTests.cs

```diff
@@ -18,21 +17,28 @@ public class ExpectedMessagingExceptionTests
             var queueClient = QueueClient.CreateFromConnectionString(
                 TestUtility.GetEntityConnectionString(Constants.NonPartitionedQueueName));
 
-            await TestUtility.SendMessagesAsync(queueClient.InnerSender, messageCount);
-            var receivedMessages = await TestUtility.ReceiveMessagesAsync(queueClient.InnerReceiver, messageCount);
+            try
+            {
+                await TestUtility.SendMessagesAsync(queueClient.InnerSender, messageCount);
+                var receivedMessages = await TestUtility.ReceiveMessagesAsync(queueClient.InnerReceiver, messageCount);
 
-            Assert.True(receivedMessages.Count() == messageCount);
+                Assert.True(receivedMessages.Count() == messageCount);
 
-            // Let the messages expire
-            await Task.Delay(TimeSpan.FromMinutes(1));
+                // Let the messages expire
+                await Task.Delay(TimeSpan.FromMinutes(1));
```

Comment:
Well based on the way we currently do things, we can create a dedicated entity for this test. It shouldn't be a problem to add to the ARM template, when we decide to do so.